### PR TITLE
feat(react-kit): multi selectbox

### PR DESCRIPTION
**Pull request by dramoturg**

### CODE REVIEW

**Comment by sutarmin** [comment]
@dramoturg any plans to continue working on this one?

File: packages/react-kit/src/components/Selectbox/Selectbox.tsx [hunk 1]
```diff
@@ -205,20 +238,20 @@ class RawSelectbox extends React.Component<TFullSelectboxProps, TSelectboxState>
 					onRequestClose={this.onPopoverRequestClose}
 					anchor={this.anchor}>
 					<Menu onItemSelect={this.onItemSelect} theme={menuTheme}>
-						{React.Children.map(children, this.wrapItem) as any}
+						{React.Children.map(children, this.wrapItem(values)) as any}
```
[the remark this answers is not in the document]
Comment (dramoturg) [reply packages/react-kit/src/components/Selectbox/Selectbox.tsx]: wrapItem will not work without `any`.

**Comment by sutarmin** [comment]
Nice work, thanks, guys! Is it a breaking change for us?

**Comment by dramoturg** [comment]
> Nice work, thanks, guys! Is it a breaking change for us?

**Comment by dramoturg** [comment]
> > Nice work, thanks, guys! Is it a breaking change for us?

not. i  didn't change  the old behavior.  now you can pass array values instead single value

**Comment by raveclassic** [comment]
What about `master`?

**Comment by sutarmin** [comment]
> What about `master`?

Good catch! Opened the PR

**Pull request merged**